### ISSUE TITLE
Add plot function for MOEAs

### ISSUE THREAD

**Issue by jakobbossek** [jakobbossek/ecr#168]
We need a nice function to visualize the Pareto front (and/or the Pareto set).
Use ggplot2 by default for 2D and offer plotly/scatterplot3d for 3D visualization.

**Comment by berndbischl** [comment]
would that operate on the optpath? paramhelpers has that to some extend

**Comment by jakobbossek** [comment]
No. The package does not manage the population within the optpath; it is optional for logging.
Maybe in v2.

**Comment by berndbischl** [comment]
But if you can optionally log during the algorithm into the path you can surely do that at the end? Which us also non performance critical? 

And I don't really see the difference between what we would like to see in visualization of fronts compared to what you want to see. So sounds like completely duplicated code.
